install_config/aggregate_logging.adoc

```diff
@@ -780,11 +780,150 @@ ifdef::openshift-enterprise[]
 You can use `oc edit dc/logging-fluentd` to update your Fluentd configuration,
 making sure to first scale down your number of replicas to zero before editing
 the deployment configuration.
-enddef::openshift-enterprise[]
+endif::openshift-enterprise[]
 
 [NOTE]
 ====
 If you are not using the provided Kibana and Elasticsearch images, you will not
 have the same multi-tenant capabilities and your data will not be restricted by
 user access to a particular project.
 ====
+
+[[configuring-curator]]
+== Configuring Curator
+
+ifdef::openshift-enterprise[]
+[NOTE]
+====
+With Aggregated Logging version 3.2.1, Curator is available for use as Tech Preview.
+To start it, after completing an installation using the 3.2.1 Deployer, scale up
+the Curator deployment config that was created (defaults to zero replicas).
```

Comment:
Something to cut n paste here would be nice.

    $ oc scale dc/logging-curator --replicas=1

This hopefully heads off the question "how high should we scale curator?"

Also, it's probably worth mentioning that if you have an ops deployment, there's a separate curator for it.